### PR TITLE
modify to add capability to read both Ginan and Bernese troposphere .…

### PR DESCRIPTION
**Pull request by umma-zannat**
to support both Ginan and Bernese tropospheric files (`.tro` files)

### CODE REVIEW

**Review by ronaldmaj — COMMENTED** [review]
Looks like some good changes coming in here to expand `.tro` reading 
- Do you have example `.tro` files we can test the new function on?

**Comment by umma-zannat** [comment]
The other format for troposhere is `cost` file. `.tro` file is basically a sinex formatted file.

**Comment by umma-zannat** [comment]
`    trop_sinex:                                 # Troposphere SINEX files are used to export troposhere products, such as ZTD and delay gradients
        output: true                           # Enable data exporting to troposphere SINEX file
        const_code:                                     # Troposphere SINEX const code
        directory:      "<OUTPUTS_ROOT>"                # Directory to export troposphere SINEX file
        filename:       "<CONFIG>-<LOGTIME>.tro" # Troposphere SINEX filename
        obs_code:       P                               # Troposphere SINEX observation code
        sol_type:       "Solution parameters"           # Troposphere SINEX solution type
        sources:        [KALMAN]                        # Source for troposphere delay data - KALMAN, etc
        version:        2                               # Troposphere SINEX version

    # log:`

File: gnssanalysis/gn_io/trop.py [hunk 1]
```diff
@@ -9,41 +9,70 @@
 from .. import gn_io as _gn_io
 
 
-def _read_tro_solution(path: str, recenter: bool = True) -> _pd.DataFrame:
+def _read_tro_solution(path: str, recenter: bool = True, trop_mode="Ginan") -> _pd.DataFrame:
+    """For backwards compatibility"""
```
Comment (ronaldmaj) [inline gnssanalysis/gn_io/trop.py:13]: Docstring should be one-line sphinx (missing the description of input parameters, errors raise, what's returned, etc.: `:param` `:return`)
https://github.com/NilsJPWerner/autoDocstring/blob/master/docs/one-line-sphinx.md

File: gnssanalysis/gn_io/trop.py [hunk 2]
```diff
@@ -9,41 +9,70 @@
...
+    return read_tro_solution(path, recenter=recenter, trop_mode=trop_mode)
+
+
+def read_tro_solution(path: str, recenter: bool = True, trop_mode="Ginan") -> _pd.DataFrame:
     """Parses tro snx file into a dataframe.
```
Comment (ronaldmaj) [inline gnssanalysis/gn_io/trop.py:18]: One-line sphinx docstring (missing the description of input parameters, errors raise, what's returned, etc.: `:param` `:return`)

**Review by ronaldmaj — CHANGES_REQUESTED** [review]
Again, looks good, couple docstrings that need to be finished off, but I think once you've got the unittests that Seb suggested in, this should be good to go

**Review by ronaldmaj — COMMENTED** [review]
Thanks heaps for putting through the `docstring` additions :pray: 

I've noticed you now have two `read_tro_solution` functions - is this intentional?
There is `_read_tro_solution` and `read_tro_solution`

**Comment by umma-zannat** [comment]
`If no one uses the function with underscore I can delete it. I personally prefer not to use underscore. `

**Review by ronaldmaj — APPROVED** [review]
Unit tests are a great addition - can use that instead of testing manually. Happy for this to go in :+1:

File: gnssanalysis/gn_io/trop.py (hunk 1 above)
Comment (ronaldmaj) [reply gnssanalysis/gn_io/trop.py:13]: This function is up for deletion if no one uses it (as there is the `read_tro_solution` function below this

**Pull request merged**